### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by glyg**

### CODE REVIEW

**Comment by glyg** [comment]
Hello @sophietheis do you have a bit of time to review this and test your code from this branch?

Thank you :)

G

**Comment by sophietheis** [comment]
Hello, 
Sure, I'm trying to take care of it this week. :) 
Sophie

**Comment by glyg** [comment]
To use the dev branch:

```sh
git fetch
git switch dev
pip install .[dev] # editable install is broken don't use -e
make tests
```

And then your code

Thanks again!

File: src/tyssue/geometry/planar_geometry.py [hunk 1]
```diff
@@ -42,6 +44,25 @@ def update_areas(sheet):
...
 
+    # @staticmethod
+    # def update_repulsion(sheet):
+    #     # Create globale grid
+    #     grid = np.mgrid[np.min(sheet.vert_df['x']) - 0.1:np.max(sheet.vert_df['x']) + 0.1:0.1,
+    #            np.min(sheet.vert_df['y']) - 0.1:np.max(sheet.vert_df['y']) + 0.1:0.1]
+    #     face_repulsion = gaussian_repulsion(grid, sheet)
+
+    #     sheet.vert_df['v_repulsion'] = 0
+    #     sheet.vert_df['grid'] = 0
```
Comment (sophietheis) [inline src/tyssue/geometry/planar_geometry.py:55]: If you comment this method due to pandas error, it is possible to fix it with that : 
```
sheet.vert_df['v_repulsion'] = sheet.vert_df['v_repulsion'].astype(object)
sheet.vert_df['grid'] = sheet.vert_df['grid'].astype(object)
```

**Review by sophietheis — COMMENTED** [review]
Do you want to keep "lineage" stuff ? I added a `lineage` attribute to `Epithelium`, which is a networkx digraph, to keep track of lineage.  There is no notion of time. It only takes cell division into account, I don't keep track of cell disappearance. 

I got this error at the first install :
 Screenshot from 2024-01-24 08-41-04
I initiated submodule pybind11 and it fixed the problem. 

We probably need to find a better way to manage the repulsion grid as it raises pandas error when pandas is updated... I don't know what a better method would be.

It seems that the last version of jupyter notebook (7.0.6) doesn't work with ipyvolume (do not work with other widget either), nothing is display. I haven't found a solution, so I switched back to version 6.5.4, and it works well.

File: src/tyssue/geometry/planar_geometry.py (hunk 1 above)
Comment (glyg) [reply src/tyssue/geometry/planar_geometry.py:55]: Thanks a lot Sophie
can you push that fix on this branch (I think you have the right to do so)?

If you have a look at the tests, I commented out some of yours, could you put them back?

Thanks a lot!

G

**Comment by sophietheis** [comment]
I can't push because of this error:
image
I hav'nt figure out how to fix it, what I have tried doesn't work :cry: 
I run a git clean, I thought the problem was modules pybind11. Even without it I can't push. 

If you have an idea how to fix it. Otherwise, I will clone the repo in the hope that would fix the problem :crossed_fingers:

**Comment by glyg** [comment]
strange .. pybind11 _should_ not be necessary any more. The other submodule is notebooks.

There is this METADATA error I need to track down, scikit-build has problems. I'll look into it

**Comment by sophietheis** [comment]
Well I don't really understand, I remove everything about pybind11, now install work; but I still can"t push. I think there is something weird/broken in my git config for this repo because I have no issue with my other repository. 
I redid it with a new clone. Now I can push, and to fix the install I run : git clean -fd

**Comment by glyg** [comment]
great thanks!

**Comment by glyg** [comment]
Hi @sophietheis can you approve the PR by doing a review (in the "Files Changes" tab)?

Thanks, I'll merge following that

**Comment by glyg** [comment]
never mind, I'll force the merge

**Pull request merged**